docs/parser.md

```diff
@@ -3,4 +3,91 @@ title: "Parser"
 weight: 90
 ---
 
->This document is under construction.
\ No newline at end of file
+## Parser
+
+AsyncAPI Parser is a package used to parse and validate the [AsyncAPI documents](asyncapi-document.md) in your Node.js or browser application. These documents can be either in YAML or JSON format.
+
+Parser validates these documents using dedicated schema-supported plugins. These supported schemas are:
+
+- AsyncAPI schema (no plugin needed)
+- OpenAPI schema
+- JSON schema
+- Avro schema
+- RAML data-type schema
+
+Parser allows the template developer to also easily access schemas provided in all of the above supported formats. This is because the JavaScript parser converts all of them into JSON schema.
```

Comment:
```suggestion
The Parser allows the template developer to easily access schemas provided in the above supported formats. This is because the JavaScript parser converts all of them into JSON schema.
```